docs/library/spi_engine/spi_engine_interconnect.rst

```diff
@@ -5,9 +5,9 @@ SPI Engine Interconnect Module
 
 .. hdl-component-diagram::
 
-The SPI Engine Interconnect module allows connecting multiple
-:ref:`spi_engine control-interface` masters to a single
-:ref:`spi_engine control-interface` slave.
+The :git-hdl:`SPI Engine Interconnect <library/spi_engine/spi_engine_interconnect>`
+allows connecting multiple :ref:`spi_engine control-interface` controllers to a single
```

Comment:
Either change 'controllers' to 'managers' here, or leave it as is here and change the 'manager' on other places to 'controller' (applies everywhere we mention the spi_engine control-interface)